lib/orderbook/OrderBook.ts

```diff
@@ -857,6 +863,25 @@ class OrderBook extends EventEmitter {
       }, requestPacket.header.id));
     }
   }
+
+  /**
+   * Returns maximum transfer amount for a swap operation for the given outbound currency.
+   * @param outboundCurrency the outbound currency during swap
+   */
+  private getLimit = (outboundCurrency: string): any => {
```

Comment:
This method should be unnecessary with the change above to replace the `enum` with a plain object. Just give that object a `{ [currency: string]: number }` type and we can refer to the `limits` object that we import directly like `limits[currency]`.